packages/plugin-figma-blade-coverage/src/utils/sendAnalytics.ts

```diff
@@ -0,0 +1,40 @@
+import { incrementTotalUseCountAsync } from '@create-figma-plugin/utilities';
+
+export const sendAnalytics = async ({
+  eventName,
+  properties,
+}: {
+  eventName: string;
+  properties: object;
```

Comment:
let's add proper types here for `properties` which things we accept/send

Reply:
honestly not sure about the properties right now we would want to send. since its a generic properties object i feel its okay to send anything and everything we pass to this object?